### ISSUE TITLE
Verify domain in e2e test after #150

### ISSUE THREAD

**Issue by jellonek** [Mirantis/virtlet#152]
In #150 I removed domain verification by it's name, after changing it from static value to generated `containerId`.
We can have old behavior using something similar to:
```
DOMAIN_NAME="$(kubectl get pod virtlet-example-cirros -o 'jsonpath={.status.containerStatuses[0].containerID}')"
```

Low priority.

**Comment by jellonek** [comment]
Outdated.
